config/admin_ranks.txt

```diff
@@ -47,3 +47,13 @@ Host			= +EVERYTHING *EVERYTHING
 
 Head Coder      = +EVERYTHING
 Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+
+
+Aspirant 	= +ADMIN +BAN +REJUV
+Knight 		= +ADMIN +BAN +REJUV +STEALTH +FUN +POSSESS +SOUND +SPAWN
```

Comment:
These need +VV

Reply:
no, the admin candidate rank does not need varedit. compano stated that knights shouldn't get VV under the clause they're likely to fuck something up after we stated paladins are very capable of doing the same thing anyways